### PR TITLE
Update Chef Infra Server Release Notes for 13.1.13

### PR DESCRIPTION
**Pull request by jobsolete**
### Description

This updates release notes documentation for Chef Infra Server version 13.1.13

### Definition of Done

Documentation accurately portrays information from [release announcement in discourse](https://discourse.chef.io/t/chef-infra-server-13-1-13-released/16282).

### Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

Signed-off by: Josh O'Brien jobrien@chef.io

### CODE REVIEW

File: chef_master/source/release_notes_server.rst [hunk 1]
```diff
@@ -5,6 +5,44 @@ Release Notes: Chef Infra Server 12.0 - 13.0.11
...
+Improvements/Bug Fixes
+-----------------------------------------------------
+
+* ``_status`` endpoint reports healthy even if ``data_collector`` is down not causing unnecessary failovers 
+* Data collector proxy-header X-Forwarded for is set as expected
+* ``chef-server-ctl`` is no longer installed in the user path only the appbundled version is installed in the user path
+* Fixes issue with Chef Support Zendesk sign-ins when a first name is not set in Hosted Chef
+* Added support for running the Chef Infra Server on Red Hat Enterprise Linux 8
+* Improvements to ``chef-server-ctl gather-logs``
+ * Add AWS to known platforms
```
Comment (IanMadd) [inline chef_master/source/release_notes_server.rst:21]: RST requires a space before a nested bullet point item
```suggestion

 * Add AWS to known platforms
```

File: chef_master/source/release_notes_server.rst [hunk 2]
```diff
@@ -5,6 +5,44 @@ Release Notes: Chef Infra Server 12.0 - 13.0.11
 
 Chef Server acts as a hub for configuration data by storing cookbooks, the policies that are applied to nodes, and metadata that describes each registered node that is managed by the chef-client.
 
+What's New in 13.1.13
+=====================================================
+
+Improvements/Bug Fixes
+-----------------------------------------------------
+
+* ``_status`` endpoint reports healthy even if ``data_collector`` is down not causing unnecessary failovers
```
Comment (IanMadd) [inline chef_master/source/release_notes_server.rst]:
```suggestion
* The ``_status`` endpoint now reports healthy even if the ``data_collector`` is down which will no longer cause unnecessary failovers.
```

File: chef_master/source/release_notes_server.rst [hunk 3]
```diff
@@ -5,6 +5,44 @@ Release Notes: Chef Infra Server 12.0 - 13.0.11
...
 
+What's New in 13.1.13
+=====================================================
+
+Improvements/Bug Fixes
+-----------------------------------------------------
+
+* ``_status`` endpoint reports healthy even if ``data_collector`` is down not causing unnecessary failovers 
+* Data collector proxy-header X-Forwarded for is set as expected
+* ``chef-server-ctl`` is no longer installed in the user path only the appbundled version is installed in the user path
```
Comment (IanMadd) [inline chef_master/source/release_notes_server.rst]:
```suggestion
* ``chef-server-ctl`` is no longer installed in the user path. Now only the appbundled version is installed in the user path.
```

File: chef_master/source/release_notes_server.rst [hunk 4]
```diff
@@ -5,6 +5,44 @@ Release Notes: Chef Infra Server 12.0 - 13.0.11
...
 Chef Server acts as a hub for configuration data by storing cookbooks, the policies that are applied to nodes, and metadata that describes each registered node that is managed by the chef-client.
 
+What's New in 13.1.13
+=====================================================
+
+Improvements/Bug Fixes
+-----------------------------------------------------
+
+* ``_status`` endpoint reports healthy even if ``data_collector`` is down not causing unnecessary failovers 
+* Data collector proxy-header X-Forwarded for is set as expected
```
Comment (IanMadd) [inline chef_master/source/release_notes_server.rst]:
```suggestion
* Data collector proxy-header X-Forwarded is set as expected.
```

File: chef_master/source/release_notes_server.rst [hunk 5]
```diff
@@ -5,6 +5,44 @@ Release Notes: Chef Infra Server 12.0 - 13.0.11
...
+* ``chef-server-ctl`` is no longer installed in the user path only the appbundled version is installed in the user path
+* Fixes issue with Chef Support Zendesk sign-ins when a first name is not set in Hosted Chef
```
Comment (IanMadd) [inline chef_master/source/release_notes_server.rst]:
```suggestion
* Fixed an issue with Chef Support Zendesk sign-ins when a first name is not set in Hosted Chef.
```

File: chef_master/source/release_notes_server.rst [hunk 6]
```diff
@@ -5,6 +5,44 @@ Release Notes: Chef Infra Server 12.0 - 13.0.11
...
+* Added support for running the Chef Infra Server on Red Hat Enterprise Linux 8
```
Comment (IanMadd) [inline chef_master/source/release_notes_server.rst]:
```suggestion
* Added support for running the Chef Infra Server on Red Hat Enterprise Linux 8.
```

File: chef_master/source/release_notes_server.rst [hunk 7]
```diff
@@ -5,6 +5,44 @@ Release Notes: Chef Infra Server 12.0 - 13.0.11
...
+* Improvements to ``chef-server-ctl gather-logs``
```
Comment (IanMadd) [inline chef_master/source/release_notes_server.rst]:
```suggestion
* ``chef-server-ctl gather-logs`` was updated with the following improvements:
```

File: chef_master/source/release_notes_server.rst [hunk 8]
```diff
@@ -5,6 +5,44 @@ Release Notes: Chef Infra Server 12.0 - 13.0.11
...
+ * Add AWS to known platforms
+ * Add AWS Native Chef Server info
+ * Add elasticsearch info
+ * Switched compression from bzip2 to gzip
+
+Deprecation Notice
+-----------------------------------------------------
+
+* SLES 11 is no longer supported per our [platform policy](https://docs.chef.io/platforms.html#platform-end-of-life-policy), as upstream support ended in March of this year.
```
Comment (IanMadd) [inline chef_master/source/release_notes_server.rst]:
```suggestion
* SLES 11 is no longer supported per our `platform policy </platforms.html#platform-end-of-life-policy>`_, as upstream support ended in March of this year.
```

**Review by IanMadd — CHANGES_REQUESTED** [review]
General suggestions on content, grammar, punctuation

**Comment by IanMadd** [comment]
Hi @OBrienCommaJosh  can you add your DCO signoff. Thanks!

**Comment by jobsolete** [comment]
Sorry @IanMadd! Added!

**Review by IanMadd — APPROVED** [review]

**Pull request merged**